pyomo/contrib/mindtpy/tabu_list.py

```diff
@@ -0,0 +1,19 @@
+from cplex.callbacks import IncumbentCallback
```

Comment:
In this file is where you are doing the cplex_available import. The tests are still failing because we import cplex directly and some testing computers don't have access to it.